### PR TITLE
Add SDKs and Modules section

### PR DESCRIPTION
**Pull request by asbjornu**
Add SDKs and Modules section underneath "Resources". Resolves #337. Needs some design love to have the platform logos and badges in place and looking nice.

### CODE REVIEW

File: resources/sdk-modules.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+{%- endfor %}
+
+[woocommerce]: https://woocommerce.com/
+[magento]: https://magento.com/
+[swedbank-pay-sdk-php]: https://github.com/SwedbankPay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-php-packagist-badge]: https://poser.pugx.org/swedbank-pay/swedbank-pay-sdk-php/version
+[swedbank-pay-sdk-php-packagist]: https://packagist.org/packages/swedbank-pay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-dotnet]: https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet
+[swedbank-pay-sdk-ios]: https://github.com/SwedbankPay/swedbank-pay-sdk-ios
+[swedbank-pay-sdk-android]: https://github.com/SwedbankPay/swedbank-pay-sdk-android
```
Comment (payex-dev) [inline resources/sdk-modules.md:168]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 45
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417&bid=17614668)

File: resources/sdk-modules.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+as of yet by Swedbank Pay. They are provided as a convenience to speed up your
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
```
Comment (payex-dev) [inline resources/sdk-modules.md:154]: Codacy Issue found: [[no-literal-urls] Don’t use literal URLs without angle brackets](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/data-protection.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/data-protection.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/test-data.md [hunk 4]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/test-data.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/demoshop.md [hunk 5]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/demoshop.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
+    {%- else -%} {{ repository.description }}
+    {%- endif -%} | [`{{ repository.name }}`]({{ repository.html_url }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:157]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/index.md [hunk 7]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/index.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/demoshop.md [hunk 8]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/demoshop.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 9]
```diff
@@ -0,0 +1,169 @@
...
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:155]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 10]
```diff
@@ -0,0 +1,169 @@
...
+  navigation:
+  - title: Resources
+    items:
+    - url: /resources/
+      title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
```
Comment (payex-dev) [inline resources/sdk-modules.md:13]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 11]
```diff
@@ -0,0 +1,169 @@
...
+    {%- else -%} {{ repository.description }}
+    {%- endif -%} | [`{{ repository.name }}`]({{ repository.html_url }})
+  {%- endif -%}
+{%- endfor %}
+
+[woocommerce]: https://woocommerce.com/
+[magento]: https://magento.com/
+[swedbank-pay-sdk-php]: https://github.com/SwedbankPay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-php-packagist-badge]: https://poser.pugx.org/swedbank-pay/swedbank-pay-sdk-php/version
+[swedbank-pay-sdk-php-packagist]: https://packagist.org/packages/swedbank-pay/swedbank-pay-sdk-php
```
Comment (payex-dev) [inline resources/sdk-modules.md:165]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 12]
```diff
@@ -0,0 +1,169 @@
...
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
```
Comment (payex-dev) [inline resources/sdk-modules.md:18]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 13]
```diff
@@ -0,0 +1,169 @@
...
+|   {%- if repository.topics contains 'swedbank-pay-checkout' -%} {% icon shopping_cart %}
+    {%- elsif repository.topics contains 'swedbank-pay-payments' -%} {% icon credit_card %}
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.topics contains 'episerver' -%} Episerver
+    {%- elsif repository.topics contains 'magento2' -%} Magento 2
+    {%- elsif repository.topics contains 'woocommerce' -%} WooCommerce
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:92]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md (hunk 9 above)
Comment (payex-dev) [inline resources/sdk-modules.md:155]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 14]
```diff
@@ -0,0 +1,169 @@
+---
+title: SDKs and Modules
+sidebar:
+  navigation:
+  - title: Resources
```
Comment (payex-dev) [inline resources/sdk-modules.md:5]: Codacy Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 15]
```diff
@@ -0,0 +1,169 @@
...
+    items:
+    - url: /resources/
```
Comment (payex-dev) [inline resources/sdk-modules.md:7]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 16]
```diff
@@ -0,0 +1,169 @@
...
+      title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
```
Comment (payex-dev) [inline resources/sdk-modules.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md (hunk 2 above)
Comment (payex-dev) [inline resources/sdk-modules.md:154]: Codacy Issue found: [[no-shortcut-reference-image] Use the trailing [] on reference images](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
+    - url: /resources/data-protection
```
Comment (payex-dev) [inline resources/sdk-modules.md:21]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md (hunk 6 above)
Comment (payex-dev) [inline resources/sdk-modules.md:157]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/index.md [hunk 18]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/index.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
+  {%- endif -%}
+{%- endfor %}
+
+[woocommerce]: https://woocommerce.com/
+[magento]: https://magento.com/
+[swedbank-pay-sdk-php]: https://github.com/SwedbankPay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-php-packagist-badge]: https://poser.pugx.org/swedbank-pay/swedbank-pay-sdk-php/version
+[swedbank-pay-sdk-php-packagist]: https://packagist.org/packages/swedbank-pay/swedbank-pay-sdk-php
+[swedbank-pay-sdk-dotnet]: https://github.com/SwedbankPay/swedbank-pay-sdk-dotnet
+[swedbank-pay-sdk-ios]: https://github.com/SwedbankPay/swedbank-pay-sdk-ios
```
Comment (payex-dev) [inline resources/sdk-modules.md:167]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/public-migration-key.md [hunk 20]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/public-migration-key.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/data-protection.md [hunk 21]
```diff
@@ -6,6 +6,8 @@ sidebar:
...
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/data-protection.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 22]
```diff
@@ -0,0 +1,169 @@
...
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
+    {%- else -%} {{ repository.description }}
+    {%- endif -%} | [`{{ repository.name }}`]({{ repository.html_url }})
+  {%- endif -%}
+{%- endfor %}
+
+[woocommerce]: https://woocommerce.com/
+[magento]: https://magento.com/
+[swedbank-pay-sdk-php]: https://github.com/SwedbankPay/swedbank-pay-sdk-php
```
Comment (payex-dev) [inline resources/sdk-modules.md:163]: Codacy Issue found: [[no-unused-definitions] Found unused definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

**Comment by payex-dev** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 39
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417&bid=17614668)

File: resources/test-data.md (hunk 4 above)
Comment (payex-dev) [inline resources/test-data.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 23]
```diff
@@ -0,0 +1,163 @@
...
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
+    - url: /resources/data-protection
+      title: Data Protection
```
Comment (payex-dev) [inline resources/sdk-modules.md:22]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/demoshop.md (hunk 8 above)
Comment (payex-dev) [inline resources/demoshop.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 24]
```diff
@@ -0,0 +1,163 @@
+---
+title: SDKs and Modules
+sidebar:
+  navigation:
+  - title: Resources
+    items:
+    - url: /resources/
```
Comment (payex-dev) [inline resources/sdk-modules.md:7]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/demoshop.md (hunk 5 above)
Comment (payex-dev) [inline resources/demoshop.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/data-protection.md (hunk 21 above)
Comment (payex-dev) [inline resources/data-protection.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 25]
```diff
@@ -0,0 +1,163 @@
...
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
+    - url: /resources/data-protection
```
Comment (payex-dev) [inline resources/sdk-modules.md:21]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 26]
```diff
@@ -0,0 +1,163 @@
...
+    items:
+    - url: /resources/
+      title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
```
Comment (payex-dev) [inline resources/sdk-modules.md:15]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/terminology.md [hunk 27]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/terminology.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 28]
```diff
@@ -0,0 +1,163 @@
...
+title: SDKs and Modules
+sidebar:
+  navigation:
+  - title: Resources
+    items:
+    - url: /resources/
+      title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
+    - url: /resources/test-data
```
Comment (payex-dev) [inline resources/sdk-modules.md:11]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 29]
```diff
@@ -0,0 +1,163 @@
...
+      title: Test Data
```
Comment (payex-dev) [inline resources/sdk-modules.md:12]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 30]
```diff
@@ -0,0 +1,163 @@
...
+    - url: /resources/demoshop
+      title: Demoshop
```
Comment (payex-dev) [inline resources/sdk-modules.md:14]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 31]
```diff
@@ -0,0 +1,163 @@
...
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
+    - url: /resources/data-protection
+      title: Data Protection
+    - url: /resources/public-migration-key
```
Comment (payex-dev) [inline resources/sdk-modules.md:23]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/public-migration-key.md [hunk 32]
```diff
@@ -6,6 +6,8 @@ sidebar:
...
+      title: SDKs and Libraries
```
Comment (payex-dev) [inline resources/public-migration-key.md:10]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 33]
```diff
@@ -0,0 +1,163 @@
...
+|   {%- if repository.topics contains 'swedbank-pay-checkout' -%} {% icon shopping_cart %}
+    {%- elsif repository.topics contains 'swedbank-pay-payments' -%} {% icon credit_card %}
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.topics contains 'episerver' -%} Episerver
+    {%- elsif repository.topics contains 'magento2' -%} Magento 2
+    {%- elsif repository.topics contains 'woocommerce' -%} WooCommerce
+    {%- else -%} ?
+    {%- endif -%}
+|   {% if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:92]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 34]
```diff
@@ -0,0 +1,163 @@
...
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
+    {%- else -%} {{ repository.description }}
+    {%- endif -%} | [`{{ repository.name }}`]({{ repository.html_url }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:157]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 35]
```diff
@@ -0,0 +1,163 @@
...
+      title: Introduction
+    - url: /resources/sdk-modules/
+      title: SDKs and Modules
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
```
Comment (payex-dev) [inline resources/sdk-modules.md:17]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 36]
```diff
@@ -0,0 +1,163 @@
...
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
+    {%- if repository.homepage != empty -%}[{{ repository.description }}]({{ repository.homepage }})
```
Comment (payex-dev) [inline resources/sdk-modules.md:155]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 37]
```diff
@@ -0,0 +1,163 @@
...
+    - url: /resources/test-data
+      title: Test Data
+    - url: /resources/demoshop
+      title: Demoshop
+    - url: /resources/development-guidelines
+      title: Open Source Development Guidelines
+    - url: /resources/release-notes
+      title: Release Notes
+    - url: /resources/terminology
+      title: Terminology
```
Comment (payex-dev) [inline resources/sdk-modules.md:20]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/public-migration-key.md (hunk 20 above)
Comment (payex-dev) [inline resources/public-migration-key.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/index.md (hunk 18 above)
Comment (payex-dev) [inline resources/index.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 38]
```diff
@@ -0,0 +1,163 @@
+---
+title: SDKs and Modules
+sidebar:
+  navigation:
+  - title: Resources
+    items:
+    - url: /resources/
+      title: Introduction
```
Comment (payex-dev) [inline resources/sdk-modules.md:8]: Codacy Issue found: [[list-item-spacing] Missing new line after list item](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 39]
```diff
@@ -0,0 +1,163 @@
...
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/sdk-modules.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/sdk-modules.md [hunk 40]
```diff
@@ -0,0 +1,163 @@
...
+as of yet by Swedbank Pay. They are provided as a convenience to speed up your
+development, so please feel free to play around. However, if you need support,
+please wait for a future, stable release." %}
+
+{:.table .table-striped}
+| Language | Library | Repository |
+| :------: | :------ | :--------- |
+{%- for repository in active_repositories -%}
+  {%- if repository.topics contains 'library' %}
+| ![{{ repository.language }}](https://img.shields.io/badge/{{ repository.language | downcase | url_encode }}-blue.svg) |
```
Comment (payex-dev) [inline resources/sdk-modules.md:154]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

File: resources/test-data.md [hunk 41]
```diff
@@ -6,6 +6,8 @@ sidebar:
     items:
     - url: /resources/
       title: Introduction
+    - url: /resources/sdk-modules/
```
Comment (payex-dev) [inline resources/test-data.md:9]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/SwedbankPay/developer.swedbankpay.com/pullRequest?prid=5373417)

**Comment by asbjornu** [comment]
>My only nitpick is that with all the **Unsuported** headings it looks like we do not support anything on this page, and it's only after reading the entire headers that I as a reader get a understanding that we are only planning on supporting some of these things in the future.

Yeah, I agree it may be a bit much. But I'd rather fail on the "right" side here of being too verbose in the communication here than beeing too subtle and have people miss the disclaimer altogether. This is hopefully just temporary anyway.

**Comment by cobraz** [comment]
Love it!

**Comment by asbjornu** [comment]
Thank you so much for your contributions, @cobraz! 🙏

**Pull request merged**